src/main/fc/fc_tasks.c

```diff
@@ -451,11 +451,11 @@ cfTask_t cfTasks[TASK_COUNT] = {
     },
 #endif
 
-#ifdef SONAR
-    [TASK_SONAR] = {
-        .taskName = "SONAR",
-        .taskFunc = taskUpdateSonar,
-        .desiredPeriod = TASK_PERIOD_MS(50),                 // every 70 ms, approximately 20 Hz
+#ifdef RANGEFINDER
+    [TASK_RANGEFINDER] = {
+        .taskName = "RANGEFINDER",
+        .taskFunc = taskUpdateRangefinder,
+        .desiredPeriod = TASK_PERIOD_MS(50),
```

Comment:
@martinbudden I did some math and looks like we are safe with 50ms. 58us per 1cm = 23ms.